packages/in_app_purchase/in_app_purchase_android/android/build.gradle

```diff
@@ -51,7 +51,8 @@ android {
 }
 
 dependencies {
-    implementation 'androidx.annotation:annotation:1.5.0'
+    implementation 'androidx.annotation:annotation:1.6.0'
+    implementation(platform("org.jetbrains.kotlin:kotlin-bom:1.8.0"))
```

Comment:
@reidbaker this is the line I was wondering about. 

[This link](https://youtrack.jetbrains.com/issue/KT-55297/kotlin-stdlib-should-declare-constraints-on-kotlin-stdlib-jdk8-and-kotlin-stdlib-jdk7) is where I got the idea from.

Reply:
I am not opposed to this but I think you should add the following comment. 
```
// org.jetbrains.kotlin:kotlin-bom artifact purpose is to align kotlin stdlib and related code versions. 
// See: https://youtrack.jetbrains.com/issue/KT-55297/kotlin-stdlib-should-declare-constraints-on-kotlin-stdlib-jdk8-and-kotlin-stdlib-jdk7
```